src/panels/developer-tools/service/developer-tools-service.ts

```diff
@@ -295,12 +299,14 @@ class HaPanelDevService extends LitElement {
     }
   );
 
-  private async _callService() {
+  private async _callService(ev) {
+    const button = ev.currentTarget;
     if (!this._serviceData?.service) {
       return;
     }
     try {
       await callExecuteScript(this.hass, [this._serviceData]);
+      button.actionSuccess();
```

Comment:
Move this to after the try block and add a return in the catch section.
Now it will provide no feedback for services that are expected to stop the webserver